apps/api/README.md

```
@@ -51,6 +51,16 @@ UPSTASH_REDIS_REST_TOKEN=
+## 📞 Contact
 
-Copyright 2024 Plura.
\ No newline at end of file
+- Project Link: [GitHub Repository](https://github.com/Skidgod4444/plura)
+- Discord: [Discord](https://discord.gg/plura)
+---
+
+### Contributing
+
+<a href="https://github.com/SkidGod4444/plura/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=SkidGod4444/plura" />
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)